lib/Controller/RoomController.php

```diff
@@ -219,6 +219,15 @@ protected function formatRoom(Room $room, Participant $participant = null) {
 			}
 		}
 
+		$inactiveUsers = array_filter($participants['users'], function($data) {
+			return $data['lastPing'] <= time() - 30 &&
+				$data['sessionId'] !== 0 && $data['sessionId'] !== '' && $data['sessionId'] !== '0';
+		});
+
+		foreach ($inactiveUsers as $inactiveUserId => $data) {
+			$room->leaveRoom($inactiveUserId);
```

Comment:
Needs cast to `(string)` IIRC for users with only numbers as user id.

Reply:
Also I think a "format" function is not the right place for such logic?

Reply:
>  Needs cast to (string) IIRC for users with only numbers as user id.

I will add the cast.

>  Also I think a "format" function is not the right place for such logic?

I also think that it is not the right place, but I just put it where the kick out code for guest participants already was.

Reply:
Ping, we want to do a release this week :)